### ISSUE TITLE
Plugin breaks with eslint 8.40.0

### ISSUE THREAD

**Issue by freaktechnik** [phanect/eslint-plugin-editorconfig#59]
Eslint 8.40.0 added some new fields to context, and klonaing the context runs into an error with it:
```
Occurred while linting [...]/index.js
    at validate ([...]/node_modules/eslint/lib/source-code/source-code.js:35:14)
    at new SourceCode ([...]/node_modules/eslint/lib/source-code/source-code.js:183:9)
    at klona ([...]/node_modules/klona/lite/index.js:8:10)
    at klona ([...]/node_modules/klona/lite/index.js:25:15)
    at Object.create ([...]/node_modules/eslint-plugin-editorconfig/lib/base.js:34:24)
    at createRuleListeners ([...]/node_modules/eslint/lib/linter/linter.js:869:21)
    at [...]/node_modules/eslint/lib/linter/linter.js:1039:110
    at Array.forEach (<anonymous>)
    at runRules ([...]/node_modules/eslint/lib/linter/linter.js:976:34)
    at Linter._verifyWithoutProcessors ([...]/node_modules/eslint/lib/linter/linter.js:1328:31)
```

**Comment by xicri** [comment]
Thanks for reporting! I will take a look.

**Comment by phanect** [comment]
Thanks for reporting!
This issue should be resolved by the v4.0.3 release. Can you check if it works?
If it doesn't work properly, feel free to reopen this issue.

**Comment by freaktechnik** [comment]
Thanks for the quick fix. Looking good now.
